management-agent/management-tsa-impl-v2/src/main/java/com/terracotta/management/service/impl/MonitoringServiceImplV2.java

```diff
@@ -51,6 +52,11 @@ public MonitoringServiceImplV2(ServerManagementServiceV2 serverManagementService
     return serverManagementService.getServersStatistics(serverNames, attributes);
   }
 
+  @Override
+  public ResponseEntityV2<StatisticsEntityV2> getServerStatistics(Set<String> serverNames, Set<String> attributes, UriInfo info) throws ServiceExecutionException {
+    throw new UnsupportedOperationException();
```

Comment:
Not sure I get why this is throwing UOE, or why it can't just delegate to the other existing method?

Reply:
Existing getServerStatistics method seems to handle tc-management-api(rest interface on server) which doesn't require UriInfo object.
Only the class implementations dealing with tmc/api require UriInfo. So I thought delegating to the existing method will lead to confusion.